### PR TITLE
SKFunction should return Task<string>

### PR DESCRIPTION
**Pull request by feiyun0112**
#1321 

Function 'GetMyDirectReportsEmailAsync' has an invalid signature not supported by the kernel.

### CODE REVIEW

**Comment by stephentoub** [comment]
It's supported as of https://github.com/microsoft/semantic-kernel/pull/1195.  That just needs to be merged. **cough** @shawncal  **cough** :smile:

**Comment by stephentoub** [comment]
Oh, sorry, I misread the subject.  I see now the function currently returns `Task<IEnumerable<string>>`, which my PR doesn't enable, yet.  We'd just need to decide what the semantics should be of that, and then we could enable it.

**Review by lemillermicrosoft — APPROVED** [review]

File: dotnet/src/Skills/Skills.MsGraph/OrganizationHierarchySkill.cs [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 
 using System.Collections.Generic;
```
Comment (lemillermicrosoft) [inline dotnet/src/Skills/Skills.MsGraph/OrganizationHierarchySkill.cs:3]:
```suggestion
```

File: dotnet/src/Skills/Skills.UnitTests/MsGraph/OrganizationHierarchySkillTests.cs [hunk 2]
```diff
@@ -35,10 +36,10 @@ public async Task GetMyDirectReportsEmailAsyncSucceedsAsync()
         OrganizationHierarchySkill target = new(connectorMock.Object);
 
         // Act
-        IEnumerable<string> actual = await target.GetMyDirectReportsEmailAsync(this._context);
+        string actual = await target.GetMyDirectReportsEmailAsync(this._context);
 
         // Assert
-        var set = new HashSet<string>(actual);
+        var set = new HashSet<string>(JsonSerializer.Deserialize<IEnumerable<string>>(actual));
```
Comment (lemillermicrosoft) [inline dotnet/src/Skills/Skills.UnitTests/MsGraph/OrganizationHierarchySkillTests.cs:42]:
```suggestion
        var emails = JsonSerializer.Deserialize<IEnumerable<string>>(actual);
        Assert.NotNull(emails);
        var set = new HashSet<string>(emails);
```

File: dotnet/src/Skills/Skills.UnitTests/MsGraph/OrganizationHierarchySkillTests.cs [hunk 3]
```diff
@@ -35,10 +36,11 @@ public async Task GetMyDirectReportsEmailAsyncSucceedsAsync()
...
-        IEnumerable<string> actual = await target.GetMyDirectReportsEmailAsync(this._context);
+        string actual = await target.GetMyDirectReportsEmailAsync(this._context);
 
         // Assert
-        var set = new HashSet<string>(actual);
+        var emails = JsonSerializer.Deserialize<IEnumerable<string>>(actual);
+        Assert.NotNull(emails);
         foreach (string directReportEmail in anyDirectReportsEmail)
         {
             Assert.Contains(directReportEmail, set);
```
Comment (lemillermicrosoft) [inline dotnet/src/Skills/Skills.UnitTests/MsGraph/OrganizationHierarchySkillTests.cs:46]:
```suggestion
            Assert.Contains(directReportEmail, emails);
```

**Review by lemillermicrosoft — APPROVED** [review]

**Pull request merged**